### PR TITLE
Feature/raceevent

### PR DESCRIPTION
**Pull request by maanick**
Please approve the PR

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+in three environments listed below.
+
+1) Standalone application in  any java IDE
+2) Docker container.
+3) Heroku cloud platform.
+
+## API Specifications
+
+please refer RESTDOCS End point link
+https://speedway-api.herokuapp.com/docs/index.html
```
Comment (davidgrieser) [inline README.md:13]: 😎

File: src/main/java/com/example/speedwayapi/RaceService.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+public class RaceService {
+    private final RaceRepository raceRepository;
+
+    @Autowired
```
Comment (davidgrieser) [inline src/main/java/com/example/speedwayapi/RaceService.java:13]: Not sure this is needed as you are doing the **Dependency Injection** (DI) already, but maybe this didn't work without it? It compiles and passes tests it should be fine.

**Review by davidgrieser — APPROVED** [review]
1 comment on code, but otherwise LGTM

**Review by 404-bug-not-found — APPROVED** [review]
LGTM

**Review by raghav-codes — APPROVED** [review]
LGTM

**Pull request merged**